### PR TITLE
CIF-2062 - adjust URLProvider config and Sling map

### PR DESCRIPTION
**Pull request by mhaack**
## Description

Adjust configs for new URLProvider behavior

* use URL suffix for product `url_key`/ category `url_path`
* adapt Sling mapping config example

Should be verified together with https://github.com/adobe/aem-core-cif-components/pull/602

## Related Issue

CIF-2062 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes and the overall coverage did not decrease.
- [x] All unit tests pass on CircleCi.
- [x] I ran all tests locally and they pass.

### CODE REVIEW

**Review by herzog31 — COMMENTED** [review]
I think you need to update the URLs in some of the ITs.

**Comment by mhaack** [comment]
> I think you need to update the URLs in some of the ITs. 
I fixed them already for #151 let me rebase the PR.

**Review by dplaton — APPROVED** [review]

**Comment by mhaack** [comment]
ITs are successful now as well, URLs have been adapted.

File: it.tests/src/main/java/com/venia/it/tests/CategoryPageIT.java [hunk 1]
```diff
@@ -36,7 +36,8 @@
 
     @Test
     public void testProductListPageWithSampleData() throws ClientException, IOException {
-        String pagePath = VENIA_CONTENT_US_EN_PRODUCTS_CATEGORY_PAGE + ".uid-1.html";
+
+        String pagePath = VENIA_CONTENT_US_EN_PRODUCTS_CATEGORY_PAGE + ".html/outdoor";
```
Comment (Buuhuu) [inline it.tests/src/main/java/com/venia/it/tests/CategoryPageIT.java:40]: minor but the url does not fit the urlprovider configuration (applies to all)

the ITs still work as this is only about resolution I guess (?)

**Review by Buuhuu — APPROVED** [review]

File: it.tests/src/main/java/com/venia/it/tests/CategoryPageIT.java (hunk 1 above)
Comment (mhaack) [reply it.tests/src/main/java/com/venia/it/tests/CategoryPageIT.java]: Fixed the urls to stay in sync with the configured url format.

**Review by dplaton — APPROVED** [review]

**Pull request merged**